cmd/commands/import.go

```diff
@@ -41,11 +42,19 @@ func NewImportCommand(c *Context) *cobra.Command {
 				options[k] = v
 			}
 
-			for _, key := range args {
-				_, err := c.signatory.Import(c.Context, vaultName, key, passCB, options)
-				if err != nil {
-					return err
-				}
+			fmt.Print("Enter secret key: ")
```

Comment:
What's the difference between `enter secret key` and `Enter Password` ?

Reply:
Explanation: https://github.com/ecadlabs/signatory/pull/267#discussion_r1080594783

Will it be apparent if the `Enter Password:` prompt is modified to `This is encrypted, enter the password:`?

Reply:
Oh I see! Yeah good idea, let's make it more explicit!

Reply:
Done